sdks/deku_go_interop/interop.go

```diff
@@ -0,0 +1,118 @@
+package deku_go_interop
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+)
+
+var chain_to_machine *os.File
+var machine_to_chain *os.File
+
+type chain_message struct {
+	Type    string
+	Payload interface{}
+}
+
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+
+func log(message string) {
+	// TODO: implement debug mode logging.
+	if false {
+		colored := fmt.Sprintf("\x1b[%dm%s\x1b[0m", 31, message)
+		fmt.Fprintln(os.Stderr, colored)
+	}
+}
+
+func read() []byte {
+	// read the number of bytes being sent
+	b := make([]byte, 8)
```

Comment:
I am not fond of one-letter variables...
Is it possible to extract `8` to a constante and use it here and in the `write`?